netman/adapters/switches/cached.py

```diff
@@ -196,11 +192,11 @@ def set_trunk_mode(self, interface_id):
 
     def set_bond_access_mode(self, bond_number):
         self.real_switch.set_bond_access_mode(bond_number)
-        self.bonds_cache[bond_number].interface.port_mode = port_modes.ACCESS
+        self.bonds_cache[bond_number].port_mode = port_modes.ACCESS
```

Comment:
Would you mind change to use ACCESS as a symbol, for standardization of the code